ai-agent/src/internal/agent-source.ts

```diff
@@ -10,10 +10,10 @@ export class DegovAgentSource {
   async refresh(fastify: FastifyInstance): Promise<void> {
     try {
       const resp = await fastify.axios.get(
-        "https://raw.githubusercontent.com/ringecosystem/degov-essential/refs/heads/main/config/agent/config.yml"
+        "https://raw.githubusercontent.com/ringecosystem/degov-essential/refs/heads/fix-timelock/config/agent/config.yml"
```

Comment:
```suggestion
        "https://raw.githubusercontent.com/ringecosystem/degov-essential/refs/heads/main/config/agent/config.yml"
```


please update it before merge